force-app/main/default/triggers/ChangeOnQuoteSentStage.trigger

```diff
@@ -0,0 +1,23 @@
+trigger ChangeOnQuoteSentStage on Opportunity (After update) {
+    List<Task> taskList = new List<Task>(); 
+    for(Opportunity oppOld : Trigger.Old) {
+        if(oppOld.StageName == 'Quote Sent')
```

Comment:
Create any class Constants that will keep all your statuses/record type names/etc. Then you will use variables from this class, instead of calling their names every time.
ex:
public class Constants {
    public static final String OPP_STAGE_QUOTE_SENT = 'Quote Sent';
}
and usage: if(oppOld.StageName == Constants.OPP_STAGE_QUOTE_SENT )

It allows you to do less work when your status change - you need to update only one reference in one class instead of searching all classes that use this stage name.